lib/src/Base/Graph/Polygon.cxx

```diff
@@ -169,4 +169,29 @@ void Polygon::load(Advocate & adv)
   adv.loadAttribute( "edgeColor_", edgeColor_ );
 }
 
+/* Builds a polygon which fills the area between two curves */
+Polygon Polygon::FillBetween(Sample const& dataX, Sample const& dataY1, Sample const& dataY2)
+{
+  return FillBetween(dataX.asPoint(), dataY1.asPoint(), dataY2.asPoint());
+}
+
+/* Builds a polygon which fills the area between two curves */
+Polygon Polygon::FillBetween(Point const& dataX, Point const& dataY1, Point const& dataY2)
```

Comment:
Same remark here regarding the use of const references instead of references of const objects

Reply:
I think both syntaxes have the same meaning.
In any case, it is not in principle possible to modify the address of a reference.
On the other hand, for pointers, we can distinguish `Type const*ptr` from `Type *const ptr`. The important syntactic modification is the position of `const` relative to `*`.

Reply:
@ariste-sarl You are right, I checked it here https://www.stroustrup.com/bs_faq2.html#constplacement.
The point is that const is at the left of type identifiers in all other places in the code (and yes, I haven't checked every line of the code so you can probably find a few exceptions). So you can see it as a coding rule in the platform.

Reply:
OK, I have done this change (in both instances).